src/components/GuideLegendaries/index.js

```diff
@@ -1,201 +1,447 @@
-        play Shadowfen) until you get Bragda at level 3+.
+        Finishers are essential in every deck. Tegor is a powerful heavy
+        finisher while carrying high value effects. Despite his abilities being
+        random, their raw value makes him very much worth the mana cost. You can
+        push this further when you play him after clearing all enemy units. This
+        guarantees that you get either the spawn or buff effect. The buff effect
+        is incredibly powerful and should be exploited whenever possible.
+        Tegor’s dragon type also synergizes with dragon decks, especially from
+        his own spawns. Tegor is the perfect option for decks that need
+        finishers.
       </p>
+
+      <h3>
+        <CardLink id='S19' />: S
+      </h3>
       <p>
-        <CardLink id='N46' />:{' '}
-        <span className='Highlight'>small yes, maybe</span>. It can be good, but
-        it is inherently random. Having it trigger its 5 damage on a 1 strength
-        unit is not a good outcome overall. If your average card level is less
-        than 3, it can have some value. Past that, other cards will be better.
+        Xuri synergizes perfectly with Swarm’s natural unit spam. Cards like{' '}
+        <CardLink id='S1' /> and other cheap cards are essential for every Xuri
+        deck. His buffs over friendly units can easily outvalue your opponent
+        and it is not hard to pull off early on. Additionally, vitality is very
+        powerful in the lower ranks since +1 strength per turn is a large
+        percentage increase compared to higher ranks. Like other dragon cards,
+        Xuri can also synergize in dragon decks that use cards like{' '}
+        <CardLink id='N6' /> and <CardLink id='S4' /> due to the powerful
+        frontline of their abilities. Xuri is definitely worth trying out and
+        can exceed in any Swarm deck.
       </p>
+
+      <h3>
+        <CardLink id='F21' />: S
+      </h3>
       <p>
-        <CardLink id='N48' />: maybe not—especially since it can no longer play{' '}
-        <CardLink id='W19' />. Perhaps if you plan to trigger{' '}
-        <CardLink id='N44' /> instead. With Gift of the Wise, you can use it one
-        turn earlier and get a free 7 strength unit, so no reason not to use it
-        in this case. With Needle Blast, you get less value, but it remains
-        alright. If planning on using <CardLink id='N29' /> instead, then it
-        gets weaker.
+        Qordia is arguably the best legendary dragon. Her egg spawns allow her
+        to apply serious pressure early game while also being able to generate
+        crazy value in long games. Not all of her eggs need to survive either to
+        find success. The eggs force the opponent to make uncomfortable plays,
+        which is always a good thing for you. Even if all the eggs were
+        destroyed or you managed to hatch only one, you still utilized 8
```

Comment:
> used

If you want to keep the word, prefer “utilised” as we use British English across the site, not American English.